src/main/java/buildcraftAdditions/reference/enums/EnumMachineUpgrades.java

```diff
@@ -26,17 +24,14 @@
 	SPEED_2("upgradeSpeed2", false),
 	SPEED_3("upgradeSpeed3", false);
 
-	private String tag;
-	private boolean multipleInstalls;
-	private ResourceLocation texture;
+	private final String tag;
+	private final boolean multipleInstalls;
+	private final ResourceLocation texture;
 
-	private EnumMachineUpgrades(String tag, boolean canBeDouble) {
+	private EnumMachineUpgrades(String tag, boolean multipleInstalls) {
 		this.tag = tag;
-		this.multipleInstalls = canBeDouble;
-		if (FMLCommonHandler.instance().getEffectiveSide() == Side.CLIENT) {
-			//DON'T REGISTER ICONS ON THE SERVER SIDE
-			this.texture = new ResourceLocation(Variables.MOD.ID, "textures/items/" + tag + ".png");
-		}
+		this.multipleInstalls = multipleInstalls;
+		texture = new ResourceLocation(Variables.MOD.ID, "textures/items/" + tag + ".png");
```

Comment:
see the comment you removed, check the side we don't need/want to register the texture at the server side